src/main/java/domain/pages/HomePage.java

```diff
@@ -10,46 +12,53 @@
     Boolean result;
     String regex = "^[0-9]{5}(?:-[0-9]{4})?$";
 
-    @FindBy (xpath = "(//a[@href = '/representatives'])[1]")
+    @FindBy (xpath = "//a[@title='Home']")
+    private WebElement homePageLogo;
+
+    @FindBy(xpath = "(//a[@href = '/representatives'])[1]")
     private WebElement representativesLink;
 
-    @FindBy (xpath = "//button[@value='Look up'][1]")
+    @FindBy(xpath = "//button[@value='Look up'][1]")
     private WebElement lookUpZipCodeButton;
 
-    @FindBy (xpath = "//input[@id='header-find-rep-zip-code']")
+    @FindBy(xpath = "//input[@id='header-find-rep-zip-code']")
     private WebElement inputZipCode;
 
-    @FindBy (xpath = "//*[contains(text(), 'You submitted an invalid Zip Code')] ")
+    @FindBy(xpath = "//*[contains(text(), 'You submitted an invalid Zip Code')]")
```

Comment:
avoid using *

Reply:
Changed to 
//form[contains(text(), 'Enter a zip code:')]